src/util/getIdentifiers.js

```diff
@@ -2,12 +2,23 @@ const matchToKartoffel = require('./matchToKartoffel');
 const fn = require('../config/fieldNames');
 
 /**
- * Returns the identifiers for the raw data record
+ * Returns the identifiers for the raw data record,
+ * if it didn't already passed through matchToKartoffel,
+ * otherwise just return them.
  * 
- * @param { object } record - raw data object 
+ * @param { Object } record - raw data object 
  * @param { string } dataSource - which data source
+ * 
+ * @returns { Object } - found identifiers
  */
 const getIdentifiers = async (record, dataSource) => {
+    if (record.personalNumber || record.identityCard) {
```

Comment:
Check by specific parameter

Reply:
add comment at updateHandler that explain why net the condition with "needMatchToKartoffel"